src/foundry/client-esm/nue/tour.d.mts

```diff
@@ -1,9 +1,276 @@
+   * Set-up operations performed before a step is shown.
+   * @virtual
+   */
+  protected _preStep(): Promise<void>;
+
+  /**
+   * Clean-up operations performed after a step is completed.
+   * @virtual
+   */
+  protected _postStep(): Promise<void>;
+
+  /**
+   * Renders the current Step of the Tour
+   */
+  protected _renderStep(): Promise<void>;
+
+  /**
+   * Reloads the Tour's current step from the saved progress
+   * @internal
+   */
+  _reloadProgress(): void;
+}
+
+declare namespace Tour {
+  /** Tour configuration data */
+  interface Config {
+    /** The namespace this Tour belongs to. Typically, the name of the package which implements the tour should be used */
+    namespace: string;
+
+    /** A machine-friendly id of the Tour, must be unique within the provided namespace */
+    id: string;
+
+    /** A human-readable name for this Tour. Localized. */
+    title: string;
+
+    /** The list of Tour Steps */
+    steps: Tour.Step[];
+
+    /** A human-readable description of this Tour. Localized. */
+    description?: string;
+
+    /** A map of localizations for the Tour that should be merged into the default localizations */
+    localization?: Localization.Translations; // TODO: Verify
+
+    /** Whether the Tour is restricted to the GM only. Defaults to false. */
+    restricted?: boolean;
+
+    /** Whether the Tour should be displayed in the Manage Tours UI. Defaults to false. */
+    display?: boolean;
+
+    /** Whether the Tour can be resumed or if it always needs to start from the beginning. Defaults to false. */
+    canBeResumed?: boolean;
+
+    /** A list of namespaced Tours that might be suggested to the user when this Tour is completed. The first non-completed Tour in the array will be recommended. */
+    suggestedNextTours?: string[];
+  }
+
+  /** A step in a Tour */
+  interface Step {
+    /** A machine-friendly id of the Tour Step */
+    id: string;
+
+    /** The title of the step, displayed in the tooltip header */
+    title: string;
+
+    /** Raw HTML content displayed during the step */
+    content: string;
+
+    /** A DOM selector which denotes an element to highlight during this step. If omitted, the step is displayed in the center of the screen. */
+    selector?: string;
+
+    /** How the tooltip for the step should be displayed relative to the target element. If omitted, the best direction will be attempted to be auto-selected. */
+    tooltipDirection?: TooltipManager.TOOLTIP_DIRECTIONS;
+
+    /** Whether the Step is restricted to the GM only. Defaults to false. */
+    restricted?: boolean;
+
+    /** Activates a particular sidebar tab. Usable in `SidebarTour` instances. */
+    sidebarTab?: string;
+
+    /** Activates a particular canvas layer and its respective control group. Usable in `CanvasTour` instances */
+    layer?: string;
+
+    /** Activates a particular tool. Usable in `CanvasTour` instances. */
+    tool?: string;
+  }
+
+  type STATUS = ValueOf<typeof Tour.STATUS>;
```

Comment:
Make this `Status` please.
1. The Foundry's type's name is `TourStatus`
2. We try not to break PascalCase convention. It has happened but for stuff like enums of static members with `SCREAMING_SNAKE_CASE`.